### PR TITLE
access levels

### PR DESCRIPTION
**Pull request by Alina-dev-front**

### CODE REVIEW

File: mrscake/src/App.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```
Comment (Parul76) [inline mrscake/src/App.js:1]: why you use this useeffect, usestate? I just want to know functionality.

File: mrscake/src/App.js [hunk 2]
```diff
@@ -11,12 +11,30 @@ import ContactUs from './components/ContactUs';
 import ProductTable from './components/ProductPage';
 import ResetPassword from './components/ResetPassword';
 import NavBar from './components/NavBar';
-//import CarouselControlled from "./components/Carrousel";
```
Comment (Parul76) [inline mrscake/src/App.js:14]: Why we comment this carousel, may be we are not getting images due to this component comment out?

File: mrscake/src/App.js (hunk 1 above)
Comment (Alina-dev-front) [reply mrscake/src/App.js:1]: this gonna be applied to know who(role) is logged in, so to adopt website for his functionality

File: mrscake/src/App.js [hunk 3]
```diff
@@ -11,12 +11,30 @@ import ContactUs from './components/ContactUs';
...
 import CheckoutForm from "./components/CheckoutForm";
-
+// import { USERS_API_URL } from './constants/user_api_url';
```
Comment (Parul76) [inline mrscake/src/App.js:15]: I think you added this line and comment out. We don't need this line or you forgot uncomment this line?

File: mrscake/src/App.js [hunk 4]
```diff
@@ -11,12 +11,30 @@ import ContactUs from './components/ContactUs';
...
+  //   fetch(`${USERS_API_URL}`)
+  //     .then(response => {
+  //       var dbResponse = response.json();
+  //       return dbResponse;
+  //   })
+  //     .then(userData => this.setRole(userData.userRole))
+  //     .catch(err => console.log(err));
+  // }
+
+
```
Comment (Parul76) [inline mrscake/src/App.js:37]: why you comment out above code as you wrote it?

File: mrscake/src/components/HomePage.jsx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import CarouselControlled from "./Carrousel";
+// import CarouselControlled from "./Carrousel";
```
Comment (Parul76) [inline mrscake/src/components/HomePage.jsx]: why carousel comment ?

**Review by Parul76 — APPROVED** [review]

**Pull request merged**